kube/services/revproxy/gen3.nginx.conf/cohort-middleware-service.conf

```diff
@@ -0,0 +1,8 @@
+location /cohort-middleware/ {
+    set $proxy_service  "cohort-middleware";
```

Comment:
Do we need $csrf_check here as well?

Reply:
I was looking into Guppy and it doesn't have [one](https://github.com/uc-cdis/cloud-automation/blob/c7e0c4c3865ef1dba2959317206a2194a6644c1c/kube/services/revproxy/gen3.nginx.conf/guppy-service.conf). So, I'm not sure if I need it. :)

Reply:
I stand corrected, we don't need one.